visivo/models/model.py

```diff
@@ -1,12 +1,53 @@
-from typing import Optional
-from pydantic import Field, validator
+from typing import Any, List, Optional, Union
+from typing_extensions import Annotated
+from pydantic import Field, Discriminator, Tag
 from visivo.models.base.named_model import NamedModel
 from visivo.models.base.parent_model import ParentModel
-from visivo.models.target import DefaultTarget, Target
-from .base.base_model import generate_ref_field
+from visivo.models.target import DefaultTarget, Target, TypeEnum
+from .base.base_model import RefString, generate_ref_field
+import pandas
+import re
 
 
-class Model(NamedModel, ParentModel):
+class Model(NamedModel):
+    pass
+
+
+class RunModel(Model):
+    cmds: List[str] = Field(description="The sql used to generate your base data")
```

Comment:
Would these be commandline prompts that create a csv? Are you thinking to make it a list of commands so that multiple different lines can be run? Also how would we handle multiple files getting created?

Reply:
I guess we'd probably just use the one that corresponds to the table_name? I wonder if we should call that file name.